src/authenticate.py

```diff
@@ -41,21 +41,21 @@ def exchange_cognito_code(event: dict, cognito_code: str) -> dict:
         'redirect_uri': redirect_uri,
         'code': cognito_code,
     }
-    structlog.get_logger().msg("Validating Cognito code")
+    logger.info("Validating Cognito code")
     try:
         token_response = requests.post(
             endpointurl,
             data=post_data,
             auth=requests.auth.HTTPBasicAuth(client_id, client_secret)
         )
     except requests.exceptions.ConnectionError as e:
-        structlog.get_logger().msg("Connection error to Cognito", exception=e)
+        logger.exception({"message": "Connection error to Cognito", "exception": e})
```

Comment:
```suggestion
        logger.exception("Connection error to Cognito")
```

Ik denk dat je algemeen `message` kan weglaten?

En ik denk dat bij logger.exception je de exception ook niet expliciet moet meegeven

(het kan zijn dat hierdoor het output formaat iets anders wordt? in plaats van

```yaml
level: ERROR
message:
  message: "Connection error to Cognito"
  exception: {...}
```  
 
 krijg je
 ```yaml
level: ERROR
message:
  message: "Connection error to Cognito"
  exception: {...}
```

Reply:
tenzij je op andere plaatsen wel een nested dict wiil natuurlijk, als kan je daar ook dit van maken:

```python
logger.info("message", extra={"reply": error_response})
```